project/src/main/java/com/google/sps/servlets/EditUserServlet.java

```diff
@@ -78,6 +85,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         }else{
             user.setProperty("phone", oldInfo.getPhone());
         }
+        if(!request.getParameter("password").isEmpty()){
+            user.setProperty("password", PasswordHash.hashPassword((request.getParameter("password")).toCharArray()));
```

Comment:
what will happen if I type in an existing email address as an account but leave the password field empty? will this reset the password for the existing user?

Reply:
No, the old password will remain.